src/main/java/cbot/io/UI.java

```diff
@@ -0,0 +1,102 @@
+package cbot.io;
+
+import java.util.ArrayList;
+
+/**
+ * Manages printing to and receiving from the user, mainly via the Command Line Interface.
+ */
+public class UI {
+
+    //CHECKSTYLE.OFF: SingleSpaceSeparator
+
+    // Frequently Used Strings
+    private static final String BLANK     = "   ";
```

Comment:
I like how the commonly used Strings are all collated at the top of the code. It makes the code look a lot neater.